multires-lyrics-search/tests/test_multireslyrics.py

```diff
@@ -11,42 +11,44 @@
 
 NUM_DOCS = 100
 TOP_K = 3
-INDEX_FLOW_FILE_PATH = 'flows/index.yml'
-QUERY_FLOW_FILE_PATH = 'flows/query.yml'
+INDEX_FLOW_FILE_PATH = "flows/index.yml"
+QUERY_FLOW_FILE_PATH = "flows/query.yml"
 
 
 def config(tmpdir):
-    parallel = 2 if sys.argv[1] == 'index' else 1
+    parallel = 2 if sys.argv[1] == "index" else 1
 
-    os.environ.setdefault('JINA_MAX_DOCS', '100')
-    os.environ.setdefault('JINA_PARALLEL', str(parallel))
-    os.environ.setdefault('JINA_SHARDS', str(4))
-    os.environ.setdefault('JINA_WORKSPACE', './workspace')
-    os.makedirs(os.environ['JINA_WORKSPACE'], exist_ok=True)
+    os.environ.setdefault("JINA_MAX_DOCS", "100")
```

Comment:
better use `single quote` as is `jina style standard`

Reply:
Ah ok. This was `black`'s doing.

Reply:
keep it, we will see

Reply:
Configure black with `black -S`, which will skip the string normalization.

Reply:
I replaced all the double with single-quotes, apart from the query - expected results (where double quotes are required because there are singles within the text)